ui/src/views/WorkerManager/routes.js

```diff
@@ -1,16 +1,16 @@
 import lazy from '../../utils/lazy';
 
-const WorkerManagerViewWorkerTypes = lazy(() =>
-  import(/* webpackChunkName: 'WorkerManager.WMViewWorkerTypes' */ './WMViewWorkerTypes')
+const WorkerManagerViewWorkerPools = lazy(() =>
+  import(/* webpackChunkName: 'WorkerManager.WMViewWorkerPools' */ './WMViewWorkerPools')
 );
 const WorkerManagerViewWorkers = lazy(() =>
-  import(/* webpackChunkName: 'WorkerManager.WMViewWorkerType' */ './WMViewWorkers')
+  import(/* webpackChunkName: 'WorkerManager.WMViewWorkerPool' */ './WMViewWorkers')
```

Comment:
Should this be `./WMViewWorkerPool`?

Reply:
No, it should be `WorkerManager.WMViewWorker` - thank you for spotting this!